l10n/en/mozorg/contribute.ftl

```diff
@@ -0,0 +1,65 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/contribute/
+
+## Page heading
+
+contribute-page-title = Volunteer Opportunities at { -brand-name-mozilla }
+contribute-page-heading = Your talents are needed for a better internet
+contribute-page-intro = { -brand-name-mozilla } is a non-profit organization working to ensure the internet is open and welcoming to all. And we need your help. By joining our community, you can help drive innovation, enhance accountability and trust and make the internet a better place for everyone.
+contribute-how-heading = How you can contribute…
+
+## Contribution tasks
+
+contribute-tag-non-technical = Non Technical
+contribute-tag-technical = Technical
+contribute-tag-technical-non-technical = Technical and Non-technical
+contribute-tag-individual-event = Individual and event organization
+contribute-tag-individual = Individual
+contribute-tag-support = Support
+contribute-task-translate-heading = Translate content
+contribute-task-translate-desc = The internet is only global if it’s understood everywhere. Help us translate { -brand-name-mozilla } products and websites into your local language.
+contribute-task-mozilla-codebase-heading = Contribute to the { -brand-name-mozilla } codebase
+contribute-task-mozilla-codebase-desc = Actively improve { -brand-name-mozilla } products by contributing to a variety of development opportunities.
+contribute-task-support-desc = Help make { -brand-name-mozilla } products easy to use. Answer user’s "help" questions as part of the { -brand-name-mozilla } Support Community forums.
```

Comment:
Should probably use curly quotes around `help`

Reply:
Also, I think this should be `users’` to make it plural, though it's one of those weird possessives with the hanging apostrophe. Or we could try `answer people’s “help” questions` if we think `users’` is too clunky.